cmd/cmdcommon/cmdcommon.go

```diff
@@ -166,3 +166,11 @@ func updateFlagSet(flags ...*flag.FlagSet) {
 		pflag.CommandLine.AddGoFlagSet(f)
 	}
 }
+
+func getOperatorNamespaceFromEnv() (string, error) {
```

Comment:
why this function is defined both here and in `pkg/util/util.go` ?

Reply:
The pkg/util version does not return error